src/vstart.sh

```diff
@@ -868,6 +868,8 @@ if [ -n "$MON_ADDR" ]; then
 	CMON_ARGS=" -m "$MON_ADDR
 	COSD_ARGS=" -m "$MON_ADDR
 	CMDS_ARGS=" -m "$MON_ADDR
+	ip="$(echo $MON_ADDR|awk -F':' '{print $1}')"
+	CEPH_PORT="$(echo $MON_ADDR|awk -F':' '{print $2}')"
```

Comment:
yeah. we need to respect the port number specified in the `-m` option. but this option actually overwrites the setting specified by `-i`. put in other way, with your change, `-m` not only changes the ip of monitor, it also changes the IP address(es) of mgr. so, i'd suggest replace this option with `-p` , which allows user to specify the lower bound of port number the monitors/mgrs are using?

what do you think?

Reply:
@tchaikov . for vstart.sh, why we need -i & -m at the same time?  i think we only use one.

Reply:
@majianpeng, i am not sure why we need both of them. but by inspecting the `usage()`, `-i` is for all IP address. while `-m` is only for monitor. i guess, @yehudasa wanted to use it as a convenient way to pass `-m $MON_ADDR` option to `ceph-{mon,osd,mds}`.

Reply:
@yehudasa   Could you explain what's the purpose of '-m'?  and what's the different between '-i' and '-m'?